le_pr_delivery/delivery/urls.py

```diff
@@ -1,11 +1,14 @@
 from django.conf.urls import patterns, include, url
 from django.contrib import admin
 from . import views
+from delivery.views import *
```

Comment:
It is a little different
We can either use  from 
```from . import views``` in urls.py
 or 
```admin.autodiscover() ``` in views.py
any preferences?